Controllers/TimeWorkedAndVacationController.cs

```diff
@@ -45,48 +44,46 @@ public async Task<IActionResult> Details(int? id)
         }
 
         // GET: TimeWorkedAndVacation/Create
-        public IActionResult Create1()
+        public IActionResult Create()
         {
             return View();
         }
 
-        // POST: TimeWorkedAndVacation/Create
-        // To protect from overposting attacks, enable the specific properties you want to bind to, for 
-        // more details, see http://go.microsoft.com/fwlink/?LinkId=317598.
+
+
         [HttpPost]
         [ValidateAntiForgeryToken]
-        
-        /*public async Task<IActionResult> Create()
+        public  ActionResult Create(TotalNumberOfDaysWorked totalNumberOfDaysWorked, TotalNumberOfAvailableVacationDays totalNumberOfAvailableVacationDays)
         {
-            var TotalNumberOfDaysWorked = 0;
-            var TimeWorkedAndVacation = 0.0125 * (double) TotalNumberOfDaysWorked;
-            
+            totalNumberOfAvailableVacationDays = totalNumberOfDaysWorked * .0125;
+            totalNumberOfDaysWorked = 0;
             if (ModelState.IsValid)
             {
-                _context.Add(typeof(TimeWorkedAndVacation));
-                await _context.SaveChangesAsync();
+                _context.Add(totalNumberOfDaysWorked,totalNumberOfAvailableVacationDays);
```

Comment:
What's the difference between the _context and a repository? 
- _context is married to Entity versus repository 
- the repository CRUD methods are generic 

Did we explicitly create a repository in your other controller? 

1. I create and interface
2. Create a Repository of generic methods
3. Then have the controller methods work from the repository 


How were these two things working together within that controller?
The Repository add a layer making these methods much more generic so that the controller interacts first with the methods in the repo -- I think this is where the abstraction is happening